pkg/injector/pod_patch.go

```diff
@@ -114,7 +114,7 @@ func (i *injector) getPodPatchOperations(ar *v1.AdmissionReview,
 	}
 
 	// Keep DNS resolution outside of getSidecarContainer for unit testing.
-	placementAddress := fmt.Sprintf("%s:80", getKubernetesDNS(placementService, namespace))
+	placementAddress := fmt.Sprintf("%s:50005", getKubernetesDNS(placementService, namespace))
```

Comment:
why is the port hardcoded here but override-able in helm chart?

Reply:
value in helm-chart is used in the multiple places and we will not document it in readme.md